examples/vnet/doca_dpdk.h

```diff
@@ -148,6 +148,11 @@ struct doca_dpdk_action_l4_port_data {
 	struct rte_flow_action_set_tp l4port;
 };
 
+struct doca_dpdk_action_fwd_data {
```

Comment:
why it is not part of the union?

Reply:
it is. "doca_dpdk_action_fwd_data " is part of "doca_dpdk_action_data"
unless you mean to remove doca_dpdk_action_fwd_data  and have two actions inside doca_dpdk_action_data
1. rte_flow_action_queue 
2. rte_flow_action_port_id 

i thought it would be better to have as one action "fwd" and then specify the action (queue / port) accordingly.